code/modules/admin/holder2.dm

```diff
@@ -148,6 +173,141 @@ GLOBAL_PROTECT(href_token)
 			return TRUE //we have all the rights they have and more
 	return FALSE
 
+/// Returns whether or not the given client has a verified 2FA connection.
+/datum/admins/proc/check_2fa(client/client)
+	if (bypass_2fa)
+		return list(TRUE, null)
```

Comment:
you could put the list result in a macro to make it a bit prettier

alternatively i would say you should document this better to make explicit why you are returning the list with two elements for future readers -- it would help the legibility of this a lot i think...

Reply:
sure